src/_utils/item/Item.tsx

```diff
@@ -110,6 +112,7 @@ const Item = (props: ItemProps) => {
           'kirk-item--highlighted': highlighted,
           'kirk-item--clickable': isClickable,
           'kirk-item--hideHoverBackground': hideHoverBackground,
+          'kirk-item--disabled': disabled,
```

Comment:
following my suggestion: non need to apply this class here

Reply:
Done